tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -134,7 +135,14 @@ public function test_update_post_updates_data() {
 		$remote_post = $this->server_replica_storage->get_post( $this->post->ID );
 		$this->assertEquals( "foo bar", $remote_post->post_content );
 
-		$this->assertDataIsSynced();
+		$posts_sync_module = new Jetpack_Sync_Module_Posts();
```

Comment:
Should we mock this and `filter_post_content_and_add_links` instead?

Reply:
I will try to get rid of all sync-related test files changes in this PR, as it was an attempt to fix the publicize test which wasn't successful